src/chrome/content/rules/GMX.xml

```diff
@@ -10,45 +9,37 @@
 
 	Nonfunctional domains:
 		- (www.)gmx.ca		(→ www.gmx.com)
-
```

Comment:
What about here? It's much easier to review your changes when you change only the parts that need fixing, and don't include additional formatting changes. If you feel like the rule needs a formatting overhaul, I suggest doing that in a separate pull request.